Dockerfile

```diff
@@ -1,26 +1,41 @@
 FROM python:3.7-slim-buster
 
-ARG connectors=all
-COPY . /app
+ARG connectors=default
 
 RUN apt-get -qq update \
     && apt-get -qqy --no-install-recommends install \
         apt-utils \
         alien \
         libaio1 \
-        mongo-tools \
         mbuffer \
         wget \
+        git \
     && rm -rf /var/lib/apt/lists/* \
-    && pip install -U --no-cache-dir pip \
+    && pip install -U --no-cache-dir pip
+
+# In order to use fastsync with MongoDB Atlas we need version 100+ of mongodump
+RUN cd /tmp && \
```

Comment:
Can you make same change to this entrypoint [script](https://github.com/transferwise/pipelinewise/blob/master/dev-project/entrypoint.sh), it's the one used for dev and testing purposes.

Reply:
@Samira-El in dev env you use mongo PPA, I updated version and used package from PPA